gridfs/asynchronous/grid_file.py

```diff
@@ -1484,14 +1484,15 @@ def __init__(
     _file: Any
     _chunk_iter: Any
 
-    async def open(self) -> None:
+    async def open(self) -> AsyncGridOut:
```

Comment:
Could you explain this one?

Reply:
The current `AgnosticGridOut.open()` returns [itself](https://github.com/mongodb/motor/blob/72e68b75690825871642da363ba911531c4cc4b6/motor/motor_gridfs.py#L163), a fact which is used in the tests.  I tried making my own version of open() that wraps `GridOut.open()` but couldn't figure out how to do so.

Reply:
How about this?:
```

    _open = AsyncCommand(attr_name="open")
...
    @coroutine_annotation
    def open(self):
        return self._framework.chain_return_value(self._open(), self.get_io_loop(), self)
```

Reply:
That works, thanks!  I'll remove the change here.